docs/app/Examples/elements/Header/Content/HeaderSubheaderExample.js

```diff
@@ -0,0 +1,15 @@
+import React, {Component} from 'react';
+import {Header} from 'stardust';
+
+export default class HeaderSubheaderExample extends Component {
+  render() {
+    return (
+      <Header.H2>
+        Account Settings
+        <div className='sub header'>
+          Manage your account settings and set email preferences
+        </div>
```

Comment:
That actually be what I tried. I'll have to check. I just remember being confused the all caps thing.